bigquery/google/cloud/bigquery/table.py

```diff
@@ -463,7 +463,7 @@ def list_partitions(self, client=None):
         """
         query = self._require_client(client).run_sync_query(
```

Comment:
Possibly out of scope, but does `run_sync_query` have a DB-API style `params` substitution that we could use in lieu of `__mod__` here? Fine if not.

Reply:
Not only out of scope, but I haven't yet learned about DB-API so I don't really understand the question :)

Reply:
We can't use query parameters in the the table identifiers anyway.